docker/ci/install-pips.bash

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I separate this file out from install-python.sh to make it install only Python and PIP.  Both the dev and CI images need install-python.bash, but only the CI image needs install-pips.bash.

We might consider a more precise name like install-ai-engines.bash.